build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170105-5</CLI_NETSDK_Version>
```

Comment:
no reason, i tried to merge a conflict using the github UI and it ended up changing the ordering. does it impact anything?